solr/core/src/java/org/apache/solr/cli/RunExampleTool.java

```diff
@@ -436,7 +439,10 @@ protected void runCloudExample(CommandLine cli) throws Exception {
 
     boolean prompt = !cli.hasOption("noprompt");
     int numNodes = 2;
-    int[] cloudPorts = new int[] {8983, 7574, 8984, 7575};
+    int defaultPort =
+        Integer.parseInt(
+            cli.getOptionValue('p', SystemUtils.getEnvironmentVariable("SOLR_PORT", "8983")));
+    int[] cloudPorts = new int[] {defaultPort, defaultPort + 1, defaultPort + 2, defaultPort + 3};
```

Comment:
Yay!  Finally have port numbers that make a bit more sense...

Reply:
Yep. Of course this will ruin tutorials out there that assume something. So perhaps we should make the default as they were, but if `SOLR_HOST` or `-p` is given, we override those defaults like here?